code/__DEFINES/mobs.dm

```diff
@@ -13,6 +13,18 @@
 #define DROPLIMB_BLUNT 1
 #define DROPLIMB_BURN 2
 
+//Mob bio-types flags
+#define MOB_ORGANIC 	(1 << 0)
+#define MOB_MINERAL		(1 << 1)
+#define MOB_ROBOTIC 	(1 << 2)
+#define MOB_UNDEAD		(1 << 3)
+#define MOB_HUMANOID 	(1 << 4)
+#define MOB_BUG 		(1 << 5)
+#define MOB_BEAST		(1 << 6)
```

Comment:
Someone is invariably going to suggest that Taj/Vulp/Kidan/Vox/Skrell need `MOB_BEAST`. This is more for something that is an actual wild animal and has that generalized form---`MOB_HUMANOID` is the appropriate one to use here.

Reply:
```suggestion
#define MOB_BEAST		(1 << 6) // Do not put this on Taj/Vulp/Kidan/Vox/Skrell. Please. 
```